integration-test/src/test/java/io/lumify/it/VertexIntegrationTest.java

```diff
@@ -49,7 +49,7 @@ public void testSearchVisibleWithQueryString() throws ApiException {
         List<ClientApiVertex> vertices;
 
         // matches all visible
-        vertices = helper.vertexApi.vertexSearch(PROPERTY_QUERY_STRING, EMPTY_FILTER, null, null, null, null,
+        vertices = helper.vertexApi.vertexSearch(PROPERTY_QUERY_STRING + "*", EMPTY_FILTER, null, null, null, null,
```

Comment:
Gordon,
Wildcard is needed for this query (which queries elastic search) to work. There are other queries  (which query directly accumulo) which do not need the wildcard. We can review those tests and then decide what to do,
Thanks
Shrinivas